samples/server/petstore/kotlin-server/javalin/src/main/kotlin/org/openapitools/server/apis/StoreApi.kt

```diff
@@ -14,15 +14,15 @@ class StoreApi(private val service: StoreApiService) {
      * @param orderId ID of the order that needs to be deleted 
      */
     fun deleteOrder(ctx: Context) {
-        ctx.status(200).json(service.deleteOrder(ctx.pathParamAsClass<kotlin.String>("orderId").get()))
+        ctx.status(200).json(service.deleteOrder(ctx.pathParamAsClass<kotlin.String>("orderId").get(), ctx))
     }
 
     /**
      * Returns pet inventories by status
      * Returns a map of status codes to quantities
      */
     fun getInventory(ctx: Context) {
-        ctx.status(200).json(service.getInventory())
+        ctx.status(200).json(service.getInventory(ctx))
```

Comment:
Note the difference here: if there's another parameter, `, ` is added in front, and otherwise, we only pass the context to the service.